app/assets/stylesheets/course/assessment/submission/submissions.scss

```diff
@@ -11,4 +11,14 @@
       padding: 0 3px;
     }
   }
+
+  &.index {
+    .submissions {
+      .no-submission {
+        strong {
+          color: $red;
```

Comment:
...yikes.

The variable is fine, but then you should use a variable for the actual class that defaults to $red.